tools/wpt/runner.ts

```diff
@@ -88,20 +89,32 @@ export async function runSingleTest(
 
     const startTime = new Date().getTime();
 
+    const cmd = [
+      denoBinary(),
+      "run",
+    ];
+
+    cmd.push(
+      "-A",
+      "--unstable",
+    );
+
```

Comment:
Actually, a question here: @lucacasonato mentioned that `--inspect-brk` should be used for "run" and "update" commands, but it seems it's only handled in "run" command here. Is this expected?

Reply:
This is Deno's run command, used for executing a test in both update and run modes, both get the `inspect-brk` functionality. 
The `update` is at a higher level, to run and then update the `expectations` file.

Reply:
Ah of course, sorry, bad idea to review before the first coffee :P